cmd/notation/sign.go

```diff
@@ -120,9 +143,12 @@ Example - [Experimental] Sign an OCI artifact identified by a tag and referenced
 	cmd.SetPflagPluginConfig(command.Flags(), &opts.pluginConfig)
 	cmd.SetPflagUserMetadata(command.Flags(), &opts.userMetadata, cmd.PflagUserMetadataSignUsage)
 	cmd.SetPflagReferrersAPI(command.Flags(), &opts.allowReferrersAPI, fmt.Sprintf(cmd.PflagReferrersUsageFormat, "sign"))
+	command.Flags().StringVar(&opts.tsaServerURL, "tsa-url", "", "timestamp authority server URL")
+	command.Flags().StringVar(&opts.tsaRootCertificatePath, "tsa-root-cert", "", "filepath of timestamp authority root certificate")
```

Comment:
Do we need to mention the supported format for the root certificate?

Reply:
I think that's too detail for a description of a flag? I haven't seen a TSA root cert that cannot be read by Notation so far.